File: python/Euler_010.py
def is_prime(n):
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False
    return True


sum_of_primes = 2 + sum(i for i in range(3, 2000000, 2) if is_prime(i))
print(sum_of_primes)

File: java/Euler_010.java

public class Euler_010 {

	public static boolean isPrime(int n) {
		for (int i = 2; i <= Math.sqrt(n); i++) {
			if (n % i == 0) {
				return false;
			}
		}
		return true;
	}

	public static void main(String[] args) {
		long sum = 2L;

		for (int i = 3; i < 2000000; i += 2) {
			if (isPrime(i)) {
				sum += i;
			}
		}

		System.out.println(sum);
	}

}
